#global testerror "Err0018"

func toto() => "coucou"
const A = #include toto()